platform/darwin/scripts/generate-style-code.js

```diff
@@ -281,6 +281,30 @@ global.propertyDoc = function (propertyName, property, layerType, kind) {
             doc += `\n\nThis attribute corresponds to the <a href="https://www.mapbox.com/mapbox-gl-style-spec/#${anchor}"><code>${property.original}</code></a> layout property in the Mapbox Style Specification.`;
         }
     }
+    doc += '\n\nThis property can be set to one of the following values:\n\n' +
+           '- `MGLStyleConstantValue`\n';
+    if (property["property-function"]) {
+        doc += '- `MGLCameraStyleFunction` with an interpolation mode of:\n' +
```

Comment:
FYI, we’ve been using `*` syntax in other places, but I don’t think it matters much.